plutus-tx-plugin/test/size/max.uplc-size.golden

```diff
@@ -1 +1 @@
-36
\ No newline at end of file
+33
```

Comment:
That's nice, I wish i could see the simplified uplc here to enjoy the benefit of caseofcase+caseknown+inline!